### ISSUE TITLE
Add validation for MinRequirements

### ISSUE THREAD

**Issue by Kashomon** [GoogleCloudPlatform/k8s-cluster-bundle#26]
Now that there's a MinRequirements, there should be some validation on a Bundle that the Bundle's components in the bundle meet a specific components' min requirements.

**Comment by Kashomon** [comment]
Note: this probably depends on #25

**Comment by Kashomon** [comment]
Deferred for now since min requirements has been removed pending more thought about what this should look like.
